packages/@aws-cdk/aws-dynamodb/test/test.dynamodb.ts

```diff
@@ -1233,6 +1233,11 @@ export = {
                   "StreamArn"
                 ]
               }
+            },
+            {
+              "Action": "dynamodb:ListStreams",
+              "Effect": "Allow",
+              "Resource": "*"
```

Comment:
Why `"*"`? Like I said in the comments, the `Resource` part should be `arn:aws:dynamodb:region:account-id:table/table-name/stream/*` or `arn:aws:dynamodb:region:account-id:table/*/stream/*`.